python/lib/session.py

```diff
@@ -66,10 +75,34 @@ def create_session(self, db):
             print("Creating visit " + self.visit_label \
                   + " for CandID "  + self.cand_id)
 
+        column_names = ('CandID', 'Visit_label', 'CenterID')
+        values = (self.cand_id, self.visit_label, str(self.center_id))
+
+        if self.project_id:
+            column_names = column_names + ('ProjectID',)
+            values = values + (str(self.project_id),)
+
+        if self.subproject_id:
+            column_names = column_names + ('SubprojectID',)
+            values = values + (str(self.subproject_id),)
+        
+        current_stage = None
+        current_stage_info = db.pselect(
+            "SELECT Value FROM Config WHERE ConfigID = (SELECT ID FROM ConfigSettings WHERE Name='default_bids_stage')",
```

Comment:
Should be setting that by default to "Not Started" removing the need of the config setting and this SQL query

Reply:
Good point. Is there an added value to have a NULL vs a Not Started stage? I wonder if it would not be even better to set 'Not Started' the default value of Current_Stage in the session table. It will have the advantage to not encode the value in the code, which can cause problem if the Current_Stage structure is updated.

Reply:
@laemtl Could you bring that up at the next LORIS meeting if it was not discussed yet? I would be in favour of having the default to `Not Started`, but there might be subtleties I don't see and would not mind other people opinion on this.

@ridz1208 @driusan Thoughts regarding changing the default value of Current_Stage from `NULL` to `Not Started`?